packages/postgres-database/src/simcore_postgres_database/utils_folders.py

```diff
@@ -511,47 +570,71 @@ async def folder_create(
         )
         if entry_exists:
             raise FolderAlreadyExistsError(
-                product_name=product_name, folder=name, parent=parent, gid=gid
+                product_name=product_name, folder=name, parent=parent, gids=gids
```

Comment:
Question: from the check above, I do not get where you use the gids to find if the entry already exists??